cmd/bundle/init.go

```diff
@@ -191,12 +191,21 @@ See https://docs.databricks.com/en/dev-tools/bundles/templates.html for more inf
 		if err != nil {
 			return err
 		}
+
+		// start the spinner
+		promptSpinner := cmdio.Spinner(ctx)
+		promptSpinner <- "Downloading the template"
+
 		// TODO: Add automated test that the downloaded git repo is cleaned up.
 		// Clone the repository in the temporary directory
 		err = git.Clone(ctx, templatePath, ref, repoDir)
 		if err != nil {
 			return err
```

Comment:
It should also stop the spinner here, otherwise it can interfere with how we do error reporting at the call site.

Reply:
Done, closing the channel early and also added a new line for presentation. The output:

```
shreyas.goenka@THW32HFW6T cli % cli bundle init https://www.a.com         
⣾ Downloading the template
Error: git clone failed: git clone https://www.a.com /var/folders/lg/njll3hjx7pjcgxs6n7b290bw0000gp/T/www.a.com-2109069192 --no-tags --depth=1: exit status 128. Cloning into '/var/folders/lg/njll3hjx7pjcgxs6n7b290bw0000gp/T/www.a.com-2109069192'...
fatal: unable to access 'https://www.a.com/': Could not resolve host: www.a.com
```